### DISCUSSION TITLE
MSSQL problems with forward slash in Domain\Username url

### DISCUSSION THREAD

**Discussion by frostspanner** [kndndrj/nvim-dbee#165] [Q&A]
From what I can tell, connecting with Windows Authentication should require a url like `sqlserver://domain\user@server . . .`, but I'm unable to get this to work in my neovim config. When I open dbee there is an error saying the url `sqlserver://domain\\user@server` (note the extra forward slash) could not be parsed. I've tried writing the url several ways in the lua config, but all attempts end up with an extra forward slash or a broken neovim config due to invalid escape character. Is there any way to make this work?

**Comment by zacharyeller13** [comment]
I know this is a year and a half later, but I was just having this issue myself today. RayJameson's related discussion comment [here](https://github.com/kndndrj/nvim-dbee/discussions/96#discussioncomment-11617015) should help.  

You basically need to escape/url-encode the connection string and then it should work.  So the backslash '\\' will become '%5C': `sqlserver://domain%5Cuser@server`.
